Lib/test/test_pdb.py

```diff
@@ -213,6 +213,11 @@ def test_pdb_basic_commands():
     BAZ
     """
 
+def reset_Breakpoint_state():
+    import bdb
+    bdb.Breakpoint.next = 1
+    bdb.Breakpoint.bplist = {}
+    bdb.Breakpoint.bpbynumber = [None]
```

Comment:
I realize that this is copied from `test_bdb.py`, but perhaps:
```suggestion
        for bp in Breakpoint.bpbynumber:
        if bp:
            bp.deleteMe()
```